configure-git/action.yml

```diff
@@ -8,4 +8,4 @@ runs:
       run: |
         git config --global user.name "github-actions[bot]"
         git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
-        git config push.default current
+        git config push.default current || true  # won't work if cwd is not git repo
```

Comment:
What does this last line do?

Reply:
Allows you to just use `git push` to push the current branch without having to specify the branch name or the remote

Reply:
I suppose `--global` could be used instead of `|| true`

Reply:
Does that mean a branch called "current" or is it whatever HEAD is pointing at?

Reply:
Whatever HEAD is pointing at - https://git-scm.com/docs/git-config#Documentation/git-config.txt-pushdefault

Reply:
Thanks for the info.